src/components/Login.js

```diff
@@ -1,51 +0,0 @@
-import React, { Component } from 'react'
-import { setPage } from "../actions/indexActions";
-import { connect } from "react-redux";
-
-export class Login extends Component {
-    constructor(props) {
-        super(props)
-        this.state = {
-            pw: '',
-            verified: false
-        }
-    }
-
-    handleChange = (e) => {
-        this.setState({ pw: e.target.value });
-    }
-
-    handleSubmit = (e) => {
-        /* replace the value of v with backend function*/
-        const isVerified = this.state.pw === 'password';
-        this.setState({ verified: isVerified });
-        this.setState({ pw: '' })
-        if (isVerified) {
-            this.props.setPage('home');
-        }
-        e.preventDefault();
-    }
-
-    render() {
-        return (
-            <div>
-                <form onSubmit={this.handleSubmit}>
-                    Enter Password<br />
-                    <input type='password' onChange={this.handleChange} value={this.state.pw} />
-                    <input type='submit' value='Submit' />
-                    {/* following paragraph element can be deleted - it's just used to demonstrate that verified is being changed*/}
-                    <p>{this.state.verified.toString()}</p>
-                </form>
-            </div>
-        )
-    }
-}
-
-const mapStateToProps = state => {
-    return {
-        page: state.pageState.page
-    };
-};
-
-export default connect(mapStateToProps, { setPage })(Login);
-
```

Comment:
was this split into `components/Authentication/Authentication.js` and `components/Authentication/Login.js`

Reply:
This is related to your other comment
We no longer use password login, so the functionality of this component is no longer needed. And now `Authenticate.js` sets a `user` object in the store, as opposed to `Login.js` setting a `page` object in the store